feature/profile/src/main/java/com/iteneum/profile/UIEventProfile.kt

```diff
@@ -1,4 +1,4 @@
-package com.iteneum
+package com.iteneum.profile
 
 /**
  * File that contains [UIEventProfile] sealed class
```

Comment:
Same here rename parameters removing **profile** in the begining of each variable name.

Reply:
Move this file to presentation layer

Reply:
Parameters renamed and file relocated!